### PR TITLE
Pint 962 silobreaker incoming feed

### PR DESCRIPTION
**Pull request by godwin-eiq**
New Silobreaker extension that makes use of the edk dev_kit

### CODE REVIEW

File: extensions/silobreaker/incoming_feeds/silobreaker_incoming_feed/utils.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        data = "POST {url}"
+        url = make_url(url, api_key, shared_key, data)
+        response = None
+
+    log_service_response(log_=log, response=response)
+    log.info("Provider made a request", method="GET", url=url)
+    try:
+        data = response.json()
+    except ValueError:
+        log.error("Transformer failed, unsuspected data type encountered")
```
Comment (petarc-eiq) [inline extensions/silobreaker/incoming_feeds/silobreaker_incoming_feed/utils.py:111]: This is not transformer, this is download.

File: extensions/silobreaker/incoming_feeds/silobreaker_incoming_feed/utils.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+
+log = structlog.get_logger(__name__)
+
+DEFAULT_PORT = 80
+
+EXTENSION_NAME = "Silobreaker"
+# TODO
+# EXTENSION_VERSION = gd("eclecticiq-extension-silobreaker").version
+# PLATFORM_VERSION = gd("eiq-platform").version
```
Comment (petarc-eiq) [inline extensions/silobreaker/incoming_feeds/silobreaker_incoming_feed/utils.py:24]: We should raise this topic at a daily standup or on #team-pint, what should we put in request headers?

File: extensions/silobreaker/incoming_feeds/silobreaker_incoming_feed/utils.py [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+import hashlib
+import hmac
+from urllib import parse
+import uuid
+from datetime import datetime
+from typing import List
+
+import requests
+import structlog
+# from pkg_resources import get_distribution as gd
```
Comment (petarc-eiq) [inline extensions/silobreaker/incoming_feeds/silobreaker_incoming_feed/utils.py:11]: Remove unused import.

File: extensions/intel471/enrichments/adversary-intelligence-enricher/requirements.txt [hunk 4]
```diff
@@ -1,2 +1,4 @@
 furl==2.1.3
-requests==2.28.1
\ No newline at end of file
+requests==2.28.1
+python-dateutil==2.8.2
+structlog==22.1.0
```
[the remark this answers is not in the document]
Comment (godwin-eiq) [reply extensions/intel471/enrichments/adversary-intelligence-enricher/requirements.txt:4]: You mean the versions?

File: extensions/silobreaker/incoming_feeds/silobreaker_incoming_feed/manifest.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
+{
+  "name": "Silobreaker Incoming Feed",
+  "extension_type": "incoming",
+  "license": "MIT license",
+  "maintainer": "PINT <pint@eclecticiq.com>",
+  "producer": "Eclectic IQ",
+  "eiq_verified": true,
+  "repository": "https://extensions.eclecticiq.com",
+  "application_path": "main.py",
```
Comment (zoranjovevski) [inline extensions/silobreaker/incoming_feeds/silobreaker_incoming_feed/manifest.json:9]: I don't see main.py in the folder, change to process.py or rename the main file.

File: extensions/intel471/enrichments/adversary-intelligence-enricher/requirements.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (zoranjovevski) [reply extensions/intel471/enrichments/adversary-intelligence-enricher/requirements.txt:4]: Why this is changing in the Intel471?

**Review by zoranjovevski — DISMISSED** [review]

**Review by zoranjovevski — APPROVED** [review]

**Pull request merged**